sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -91,15 +91,22 @@ export interface StudentFileUploaderForm {
 /**
  *  Student uploader interface
  */
-export interface StudentFileUploaderDto {
+export interface StudentFileUploaderDTO {
   submittedForm: StudentFileUploaderForm;
   associatedFiles: string[];
 }
 
 /**
  *  Student uploaded documents (i.e, FileOriginType.Student documents)
  */
-export interface StudentUploadFileDto {
+export interface StudentUploadFileDTO {
```

Comment:
Is this interface being displayed on swagger? If not, can we take this opportunity and change it to a class? Same for  `StudentFileUploaderDTO`.